tensorflow_lite_support/ios/task/audio/core/BUILD

```diff
@@ -12,7 +12,6 @@ objc_library(
         "sources/TFLFloatBuffer.h",
     ],
     module_name = "TFLFloatBuffer",
-    deps = ["//third_party/apple_frameworks:Foundation"],
```

Comment:
@khanhlvg @schmidt-sebastian These dependencies get added every time a PR is merged.
These are not needed. Their inclusion leads to Bazel build failure in my local.

Reply:
Will update our script that does the copy to GitHub. Thanks for the heads up.